features/cucumber.feature

```diff
@@ -256,3 +256,16 @@ Feature: The cucumber command
     # Passing a line number activates serial test execution
     When I run `geordi cucumber --verbose single.feature:1`
     Then the output should contain "> bin/cucumber"
+
+
+  Scenario: Invalid keys are reported once per settings file
```

Comment:
The scenario title sounds like it should ensure each settings file prints warnings, which is not the case. Please change to:

```suggestion
  Scenario: Invalid config keys are not reported twice
```